### ISSUE TITLE
ovr_text.exe top text needs updating

### ISSUE THREAD

**Issue by rsm28** [mm0zct/Oculus_Touch_Steam_Link#34]
Image attached:
https://i.imgur.com/MbNLhpf.png

This does not align with the current config information within ovr_test.cpp: https://github.com/mm0zct/Oculus_Touch_Steam_Link/blob/b897cc11f70737507999f82056f10259ee3b4cb2/ovr_test/ovr_test.cpp#L275

ovr_test.exe implies needing the following format:
n
31
Oculus_link
oculus_link
[... etc...]

ovr_test.cpp wants a different format with a host of new options that aren't listed, making editing config.dat more difficult. Recommend adjusting the flavour text at the top to compensate.

_(this caused some level of pain)_

**Comment by mm0zct** [comment]
Apologies, using the arguments is somewhat deprecated since the gui has been added. You should be able to change everything in the gui now and it will be remembered. Is there something you needed to set manually via argument still? 

I'll check and clear up the argument parsing anyway

**Comment by rsm28** [comment]
I'll claim a hefty amount of fault here- I forgot the GUI existed, lol
I was debugging an odd error - ovr_test.exe just wasn't picking up anything after a recent Meta Quest Link update, and I was trying to figure out the problem and kinda threw things at the wall, and eventually something stuck. 

Not sure if others have raised the issue, but a recent version of Meta Quest Link broke CV1 controllers for a brief period of time, and my latest MQL version was one of those versions. Ended up downgrading and everything was OK

**Comment by mm0zct** [comment]
> Not sure if others have raised the issue, but a recent version of Meta Quest Link broke CV1 controllers for a brief period of time, and my latest MQL version was one of those versions. Ended up downgrading and everything was OK

Can you summarize what you mean with quest link? I wouldn't expect quest link to work at the same time as CV1 with oculus touch link - it would probably pick up the quest controllers and headset. You would have to connect the quest with virtual desktop, alvr, or I just experimentally fixed steamvr remote streaming

**Comment by rsm28** [comment]
MQL is Meta's new name for the Oculus software - same as before, just a new
and silly name
